gittorrentd

```diff
@@ -76,7 +76,7 @@ function bpad (n, buf) {
 
 dht.on('ready', function () {
   // Spider all */.git dirs and announce all refs.
-  var repos = glob.sync('*/.git/git-daemon-export-ok')
```

Comment:
console.log('in repo ' + repo)
    repo = repo.replace(/git-daemon-export-ok$/, '')